package pl.edu.agh.hangman;

import java.io.IOException;

public class Hangman {

    public static final String[] HANGMANPICS = new String[]{
            "  +---+\n" +
                    "  |   |\n" +
                    "      |\n" +
                    "      |\n" +
                    "      |\n" +
                    "      |\n" +
                    "=========",
            "  +---+\n" +
                    "  |   |\n" +
                    "  O   |\n" +
                    "      |\n" +
                    "      |\n" +
                    "      |\n" +
                    "=========",
            "  +---+\n" +
                    "  |   |\n" +
                    "  O   |\n" +
                    "  |   |\n" +
                    "      |\n" +
                    "      |\n" +
                    "=========",
            "  +---+\n" +
                    "  |   |\n" +
                    "  O   |\n" +
                    " /|   |\n" +
                    "      |\n" +
                    "      |\n" +
                    "=========",
            "  +---+\n" +
                    "  |   |\n" +
                    "  O   |\n" +
                    " /|\\  |\n" +
                    "      |\n" +
                    "      |\n" +
                    "=========",
            "  +---+\n" +
                    "  |   |\n" +
                    "  O   |\n" +
                    " /|\\  |\n" +
                    " /    |\n" +
                    "      |\n" +
                    "=========",
            "  +---+\n" +
                    "  |   |\n" +
                    "  O   |\n" +
                    " /|\\  |\n" +
                    " / \\  |\n" +
                    "      |\n" +
                    "========"
    };

    public static void main(String[] args) throws IOException, InterruptedException {
        Api.run();

//        CheckLetters nowaGra = new CheckLetters(path, dlugoscSlowa);
//        System.out.println(Word.losujSlowo(path, sourceTypeCase, dlugoscSlowa));


//        Word word = new Word(path);
//        CheckLetters gra = new CheckLetters(word.losujSlowo());
//        UserInput input = new UserInput();
//
//        do {
//            gra.containsLetter(input.userInput());
//            gra.getShowingWord();
//            gra.getGameStatus();
//            System.out.println(gra.getShowingWord());
//            System.out.println(gra.getGameStatus());
//        } while (gra.getGameStatus() != 7);
//
////        PrintBoard actual = new PrintBoard();

    }
}
